File: DataSummitSolution/DataSummitUI/src/app/efModels/imageGrid.ts
import { Drawing } from "../drawings/models/drawing";

export class ImageGrid {

	ImageGridId: number;
	Name: string;
	WidthStart: number;
	HeightStart: number;
	Width: number;
	Height: number;
	BlobUrl: string;
	Processed: boolean;
	Type: number;
	DrawingId: number;
	Drawing: Drawing;

	constructor(imageGridId?: number, name?: string, widthStart?: number,
		heightStart?: number, width?: number, height?: number,
		blobUrl?: string, processed?: boolean, type?: number,
		drawingId?: number, drawing?: Drawing)
	{
		this.ImageGridId = imageGridId || 0;
		this.Name = name || "";
		this.WidthStart = widthStart || 0;
		this.HeightStart = heightStart || 0;
		this.Width = width || 0;
		this.Height = height || 0;
		this.BlobUrl = blobUrl || "";
		this.Processed = processed || false;
		this.Type = type || 0;
		this.DrawingId = drawingId || 0;
		this.Drawing = drawing;
	}
}

File: TemplateMapper/TemplateMapper/Models/ImageGrid.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using TemplateMapper.OCR.Consolidated;

namespace Models
{
    public class ImageGrid
    {
        public long ImageGridId { get; set; }
        public string Name { get; set; }
        public int WidthStart { get; set; }
        public int HeightStart { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public string BlobUrl { get; set; }
        public bool Processed { get; set; }
        public ImageType Type { get; set; }
        public Image Image { get; set; }
        public long DrawingId { get; set; }
        public List<Sentence> Sentences { get; set; }

        public virtual Drawing Drawing { get; set; }

        public enum ImageType
        {
            Normal = 1,
            Overlap = 2
        }

        public ImageGrid()
        {
            WidthStart = 0;
            HeightStart = 0;
            Width = 0;
            Height = 0;
            Processed = false;
        }
    }
}